app/postprocessing.py

```
@@ -63,27 +61,27 @@ def load_result_processor(config: IndexConfig) -> BaseResultProcessor | None:
 
 
 def process_taxonomy_completion_response(
-    response: Response, input: str, langs: list[str]
+    response: dict, input: str, langs: list[str]
```

Ruff (0.16.1)

[error] Function argument `input` is shadowing a Python builtin

(A002)